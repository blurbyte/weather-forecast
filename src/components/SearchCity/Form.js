// Form hidden on tablets or smaller devices

import styled from 'styled-components';

import media from '../../styles/media';

const Form = styled.form`
  position: relative;

  ${media.tablet`
    display: none;
  `};
`;

export default Form;
