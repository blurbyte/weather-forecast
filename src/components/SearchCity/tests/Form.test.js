import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Form from '../Form';

test('renders correctly', () => {
  const component = renderer.create(<Form fetchForecast={jest.fn()} />);
  expect(component.toJSON()).toMatchSnapshot();
});
