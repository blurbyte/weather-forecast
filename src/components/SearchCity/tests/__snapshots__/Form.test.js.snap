// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: relative;
}

@media (max-width:40em) {
  .c0 {
    display: none;
  }
}

<form
  className="c0"
/>
`;
